src/widgets/ActivityGroups/ui/ActivityCard/ActivityCardRestartResume.tsx

```diff
@@ -83,15 +83,14 @@ export const ActivityCardRestartResume = ({
               padding: '10px 10px',
               transition: 'all 0.2s',
               minWidth: '120px',
+              mt: 2,
```

Comment:
Do we have a decided pattern for defining styles as short form vs the long form "mt" vs "marginTop"?

Reply:
Good callout! I was just moving the `mt={2}` from below into here. But in fact, we shouldn't currently use the shorthand becuase the Material UI theme in the web was never configured properly, and I observed the shorthand values are being converted using an incorrect multiplier (something other than 10px, which is what's done in the Admin App).

The Web App was written with almost no attention given to styling best practices – it's kind of a mess. 😭